frontend/src/features/books/BookDetails.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useEffect, useState } from 'react'
-import { useParams, Link } from 'react-router-dom'
+import { useParams, Link, useNavigate } from 'react-router-dom'
 import { API_URL } from '../../constants'
 
 function BookDetails() {
-  const [title, setTitle] = useState(null)
-  const [author, setAuthor] = useState(null)
-  const [description, setDescription] = useState(null)
+  const [book, setBook] = useState(null)
   const { id } = useParams()
+  const navigate = useNavigate()
 
   useEffect(() => {
     const fetchCurrentBook = async () => {
       try {
         const response = await fetch(`${API_URL}/${id}`)
-        console.log('Response status:', response.status);
         if (response.ok) {
           const json = await response.json()
-          console.log('Response JSON:', json);
-          setTitle(json.title)       
-          setAuthor(json.author)
-          setDescription(json.description)       
+          setBook(json)
         } else {
-          const errorText = await response.text();
-          throw new Error(`Error ${response.status}: ${errorText}`);
+          const errorText = await response.text()
+          throw new Error(`Error ${response.status}: ${errorText}`)
```

Comment:
@nshelokar-vf, why we've removed `;` from here?
I see we have added `;` on some files and removed it from some of those. Is there any specific reason for this change?